src/Service.Tests/Configuration/CorsUnitTests.cs

```diff
@@ -14,158 +15,159 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Primitives;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
-using Moq;
-
-namespace Azure.DataApiBuilder.Service.Tests.Configuration
+using VerifyMSTest;
+
+namespace Azure.DataApiBuilder.Service.Tests.Configuration;
+
+/// <summary>
+/// Tests that Cors is read correctly from configuration and server is configured as expected
+/// Server configuration is verified through sending HTTP requests to generated test server
+/// and comparing expected and received header values
+///
+/// Behavior of origins:["*"] resolving to "AllowAllOrigins" vs origins:["*", "any other specific host"] not doing so
+/// is verified through testing. Only documentation found relating to this:
+/// https://docs.microsoft.com/en-us/cli/azure/webapp/cors?view=azure-cli-latest
+/// """To allow all, use "*" and remove all other origins from the list"""
+/// </summary>
+[TestClass]
+public class CorsUnitTests
+    : VerifyBase
 {
+
+    #region Positive Tests
+
     /// <summary>
-    /// Tests that Cors is read correctly from configuration and server is configured as expected
-    /// Server configuration is verified through sending HTTP requests to generated test server
-    /// and comparing expected and received header values
-    ///
-    /// Behavior of origins:["*"] resolving to "AllowAllOrigins" vs origins:["*", "any other specific host"] not doing so
-    /// is verified through testing. Only documentation found relating to this:
-    /// https://docs.microsoft.com/en-us/cli/azure/webapp/cors?view=azure-cli-latest
-    /// """To allow all, use "*" and remove all other origins from the list"""
+    /// Verify correct deserialization of Cors record
     /// </summary>
-    [TestClass]
-    public class CorsUnitTests
+    [TestMethod]
+    public Task TestCorsConfigReadCorrectly()
```

Comment:
now that deserialization isn't really occuring here, could you update comment to reflect usage of filesystem/runtimeconfigloader/Verify?

Reply:
I'd argue it is happening, the `TryLoadConfig` is going to do deserialization, it's more that you don't directly control it, but it's still correct in what's happening.